src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	private Action action;
+	private Tag tag;
+	private Person person;
+
+	public Tagging(Action action, Tag tag, Person person) {
+		this.action = action;
+		this.tag = tag;
+		this.person = person;
+	}
+
+	public Action getAction() {
+		return action;
+	}
+
+	public Tag getTag() {
+		return tag;
+	}
+
+	public Person getPerson() {
+		return person;
+	}
+	
+	public String toString(){
+		return action.toString() + " " + person.getName().toString() + " " + tag.toString();
+	}
+
+	private enum Action {
+		ADDED("+"), DELETED("-");
+
+		private String value;
+
+		Action(String symbol) {
+			this.value = symbol;
+		}
+
+		public String toString() {
+			return value;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.